libnavigation-base/src/main/java/com/mapbox/navigation/base/route/DirectionsSession.kt

```diff
@@ -1,8 +1,12 @@
 package com.mapbox.navigation.base.route
 
 import com.mapbox.geojson.Point
+import com.mapbox.navigation.base.route.model.Route
 
 interface DirectionsSession {
+    var origin: Point
+    var waypoints: List<Point>
```

Comment:
Could this be nullable based on the changes introduced in `Router#getRoute` 👇 or are we envisioning this be empty in that case? If so, do we need nullable in `Router#getRoute`?

Reply:
it's outdated